src/modules/components/address/addressInput/addressInput.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { useState } from 'react';
+import { AddressInput, type IAddressInputProps, type IAddressInputResolvedValue } from './addressInput';
+
+const meta: Meta<typeof AddressInput> = {
+    title: 'Modules/Components/Address/AddressInput',
+    component: AddressInput,
+    tags: ['autodocs'],
+    parameters: {
+        design: {
+            type: 'figma',
+            url: 'https://www.figma.com/file/P0GeJKqILL7UXvaqu5Jj7V/v1.1.0?type=design&node-id=8192-18146&mode=design&t=VfR81DAQucRS3iGm-4',
+        },
+    },
+};
+
+type Story = StoryObj<typeof AddressInput>;
+
+const ControlledComponent = (props: IAddressInputProps) => {
+    const [value, setValue] = useState<string>();
+    const [addressValue, setAddressValue] = useState<IAddressInputResolvedValue>();
+
+    const stringAddressValue =
+        addressValue != null ? `{ address: ${addressValue.address}, name: ${addressValue.name} }` : 'undefined';
```

Comment:
The only reason I wasn't using `JSON.stringify` was to explicitly display a `name: undefined` value when name is not defined, but I'd say that it's not really needed. I'd prefer to keep the "undefined" fallback instead of empty string when the value is not defined 🙏🏻